crates/bevy_ui/src/render/mod.rs

```diff
@@ -344,9 +344,12 @@ pub fn prepare_uinodes(
             positions[3] + positions_diff[3].extend(0.),
         ];
 
+        let rect_size_v4 = uinode_rect.size().extend(0.).extend(0.);
```

Comment:
I don't think the _v4 is necessary. Took me a while to even realize it meant it was a vec4.